qa/cephfs/overrides/prefetch_dirfrags/yes.yaml

```diff
@@ -0,0 +1,4 @@
+overrides:
+  ceph:
+    cephfs:
+      mds_dir_prefetch: true
```

Comment:
There already has the yes.yaml and no.yaml files in qa/suites/fs/thrash/workloads/overrides/prefetch_dirfrags, you should just append your option

Reply:
Done. I created a new directory